src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -72,35 +70,36 @@ export class TechRecordSummaryComponent implements OnInit, OnDestroy, OnChanges
   ) {}
 
   ngOnInit(): void {
+    let isOnInit = true;
```

Comment:
As this is a guard to run the logic once, perhaps adding `first` to the name might make it's use clearer - what do you think?

```suggestion
    let isFirstOnInit = true;
```

Reply:
`onInit` only runs once, it's the logic inside the observable which runs multiple times. The thought is that we want that logic when we are running inside the synchronous function `onInit` and if we're not, we want to run a different logic. 

Since there isn't a "second" or "third" onInit, only one, I don't think it would be qualified as "first". We're either inside the onInit lifecycle hook, or we're not.